nestkernel/mpi_manager.h

```diff
@@ -261,6 +268,49 @@ class MPIManager : public ManagerInterface
    */
   bool adaptive_spike_buffers() const;
 
+  /**
+   * Sets the recvcounts parameter of Alltoallv for communication of
+   * secondary events, i.e., the number of elements (in ints) to recv
+   * from the corresponding rank.
+   */
+  void set_recv_counts_secondary_events_in_int_per_rank( const std::vector< int >& recv_counts_in_int_per_rank );
+
+  /**
+   * Returns the recvcounts parameter of Alltoallv for communication of
+   * secondary events, i.e., the number of elements (in ints) to recv
+   * from `source_rank`.
+   */
+  size_t get_recv_count_secondary_events_in_int( const size_t source_rank ) const;
+
+  /**
+   * Returns the rispls parameter of Alltoallv for communication of
```

Comment:
Is `rispls` a typo? Looks like it should be
```suggestion
   * Returns the recvdisplacements parameter of Alltoallv for communication of
```

Reply:
indeed a typo, should be `rdispls` (https://www.open-mpi.org/doc/v4.0/man3/MPI_Alltoallv.3.php)